### PR TITLE
Dk at 02.08.01 | <Homepage> Reset to initial state when all jobs deleted

### PR DESCRIPTION
**Pull request by breighell**
https://trello.com/c/y8ccIJ1n/2210-at020801-homepage-reset-to-initial-state-when-all-jobs-deleted

### CODE REVIEW

**Review by NatalyaVasylyeva — COMMENTED** [review]
Why do you write 
homePage
      .clickNewItemSideMenuLink()
      .typeNewItemNameInputField(freestyleProjectNewName)
      .selectFreestyleProjectItem()
      .clickOkBtnAndGoFreestyleProjectConfig()
      .clickSaveBtnAndGoFreestyleProject()
      .clickHomePageLink()

instead of using createFreestyleProject from commands.js

**Review by nadilova — DISMISSED** [review]

**Review by NatalyaVasylyeva — COMMENTED** [review]
Your "it" should have the exact name as your AT card in Trello, so is the name of your pull request

**Comment by breighell** [comment]
Some naming changes made. Thanks for the review

**Review by NatalyaVasylyeva — APPROVED** [review]

**Review by ElNov686 — APPROVED** [review]

File: cypress/e2e/tests/homePage.cy.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import newItemPageData from "../../fixtures/pom_fixtures/newItemPage.json";
 import peoplePageData from "../../fixtures/pom_fixtures/peoplePage.json";
 import {configureCloudsHeader} from "../../fixtures/pom_fixtures/configureCloudsPage.json"
 import {distributedBuildsLinkPageUrl} from "../../fixtures/pom_fixtures/distributedBuildsLinkPageData.json"
+import {freestyleProjectNewName} from "../../fixtures/pom_fixtures/freestyleProjectPage.json"
```
Comment (KatyaKaterina33) [inline cypress/e2e/tests/homePage.cy.js:13]: Please remove this import(line 13).

**Pull request merged**